.travis.yml

```diff
@@ -22,3 +22,6 @@ deploy:
     branch: master
     tags: true
     condition: $TOXENV == py27
+addons:
+  hosts:
+    - ip6-localhost
```

Comment:
Looks like it just assigns it to [127.0.0.1](https://travis-ci.org/scrapinghub/frontera/jobs/109851247#L121) :open_mouth: .. so this is probably not very helpful